persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -111,6 +113,15 @@ UPDATE Conversation SET mls_last_keying_material_update= ? WHERE mls_group_id =
 selectByKeyingMaterialUpdate:
 SELECT mls_group_id FROM Conversation WHERE mls_group_state = ? AND protocol = ? AND mls_last_keying_material_update - ? <0 AND mls_group_id IS NOT NULL;
 
+updateProposalTimer:
+UPDATE Conversation SET mls_proposal_timer = COALESCE(mls_proposal_timer, ?) WHERE mls_group_id = ?;
+
+clearProposalTimer:
+UPDATE Conversation SET mls_proposal_timer = NULL WHERE mls_group_id = ?;
+
+selectProposalTimers:
+SELECT mls_group_id, mls_proposal_timer FROM Conversation WHERE mls_group_id IS NOT NULL AND mls_proposal_timer IS NOT NULL;
```

Comment:
I see that we decide which conversation is MLS conversation with filtering by mls_group_id, isn't better to filter base on the protocol field?

Reply:
I can also filter based on `protocol` but I want to keep  doing `mls_group_id IS NOT NULL`  since it makes the query return a non-optional `mls_group_id`.

Reply:
I'll add additional filter on the `protocol`